src/python/shorah_dec.py

```diff
@@ -25,6 +25,11 @@
     file of corrected reads
 '''
 
+from __future__ import print_function
+from __future__ import division
+from builtins import dict
+from future.utils import listitems
+from builtins import range
```

Comment:
@ozagordi I am not opposed to py3-only, it's just that academia/science specifically is extremely behind on the py3 transition most people I interact with in science still use py2. I thought py2 backwards compatibility as an advantage, but as you like.

How should we proceed? Ditch py2 now already? Or leave the compatibility shim for the mean time? I personally don't see the extra dependence on the future package as an issue.